zerver/lib/actions.py

```diff
@@ -918,6 +918,7 @@ def do_add_reaction(user_profile, message, emoji_name):
             'full_name': user_profile.full_name}
 
     event = {'type': 'reaction',
+             'action': 'add',
```

Comment:
This got added in master as `'op': 'add'`; you should switch your code to use that when you rebase.